Orange/widgets/data/owoutliers.py

```diff
@@ -170,7 +170,7 @@ def commit(self):
         inliers = outliers = None
         self.n_inliers = self.n_outliers = None
         if self.data is not None and len(self.data) > 0:
-            if self.data.Y.ndim > 1:
+            if self.data.Y.ndim > 1 and self.data.Y.shape[1] > 1:
```

Comment:
a) Can we use: `if len(self.data.domain.class_vars) > 1`
b) As discussed, we should find out how you managed to get a Y with shape (n,0)

Reply:
Try this: *FIle* (open Iris) -> *Transpose* -> See in *Python Script*.